llvm/lib/CodeGen/BranchFolding.cpp

```diff
@@ -90,10 +90,13 @@ namespace {
 
   /// BranchFolderPass - Wrap branch folder in a machine function pass.
 class BranchFolderLegacy : public MachineFunctionPass {
+  bool EnableTailMerge;
+
 public:
   static char ID;
 
-  explicit BranchFolderLegacy() : MachineFunctionPass(ID) {}
+  explicit BranchFolderLegacy(bool EnableTailMerge = true)
+      : MachineFunctionPass(ID), EnableTailMerge(EnableTailMerge) {}
```

Comment:
> That added the initial port to the new pm. This is now changing the pass arguments in the old PM, without the matching new PM change

Are you talking about the default argument? I see that `BranchFolderPass` already takes a `bool EnableTailMerge`: https://github.com/llvm/llvm-project/blob/b39ab7a620e1bd038d6c22a667110ad814596288/llvm/include/llvm/CodeGen/BranchFoldingPass.h#L19